models/user.go

```diff
@@ -1050,7 +1050,17 @@ func ChangeUserName(u *User, newUserName string) (err error) {
 		return fmt.Errorf("Rename user directory: %v", err)
 	}
 
-	return nil
+	sess := x.NewSession()
```

Comment:
It would be better to move L1053-L1061 before directory rename to minimize possiblity of having one thing done and other failing